CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## 0.3.5
+
+### Enhancements
+
+- Add support for `--allow-insecure-host` (aliased to `--trusted-host`) ([#6591](https://github.com/astral-sh/uv/pull/6591))
+- Read requirements from `requires.txt` when available ([#6655](https://github.com/astral-sh/uv/pull/6655))
+- Respect `tool.uv.environments` in `pip compile --universal` ([#6663](https://github.com/astral-sh/uv/pull/6663))
+- Use relative paths by default in `uv add` ([#6686](https://github.com/astral-sh/uv/pull/6686))
+
+### CLI
+
+- Improve messages for empty solves and installs ([#6588](https://github.com/astral-sh/uv/pull/6588))
```

Comment:
Should we just collapse this section into "Enhancements"? What's the difference?

Reply:
Done